tests/compute/loop-unroll.slang

```diff
@@ -1,7 +1,15 @@
 //TEST(compute):COMPARE_COMPUTE:
+//TEST(compute):COMPARE_COMPUTE:-dx12 
+//TODO(JS): This test fails with a crash in CreateComputePipelineState, so disabled for now
+//DISABLE_TEST(compute):COMPARE_COMPUTE:-dx12 -use-dxil
+//TEST(compute):COMPARE_COMPUTE:-cpu
+//TEST(compute):COMPARE_COMPUTE:-cuda
+// Note VK output is not loop unrolled
+//TEST(compute):COMPARE_COMPUTE:-vk
 
-//TEST_INPUT:ubuffer(data=[0 1 2 3], stride=4):out
-//TEST_INPUT:ubuffer(data=[1 2 3 0], stride=4):
+//TEST_INPUT:ubuffer(data=[0 0 0 0], stride=4):out, name buffers[0]
+//TEST_INPUT:ubuffer(data=[0 1 2 3], stride=4):name buffers[1]
+//TEST_INPUT:ubuffer(data=[1 2 3 0], stride=4):name buffers[2]
```

Comment:
As discussed elsewhere, I suspect the root cause of the crash in the D3D12 + DXIL case is that these `//TEST_INPUT` lines are effectively declaring three different descriptor ranges (each of size one) while the generated DXIL is compiled to expect/require a single resource range of size three.

I'm not sure why that kind of issue would lead to a crash rather than some kind of validation error message, so maybe I'm mistaken and something more sinister is going on.